src/main/java/com/vinylteam/vinyl/dao/ShopDao.java

```diff
@@ -8,4 +8,6 @@
 
     List<Shop> getManyByListOfIds(List<Integer> ids);
 
+    List<Shop> getAll();
```

Comment:
findAll() - when we work with db, we "find" - to make it different from "get" within application, as far as I understand.